pkg/appgateserver/cmd/cmd.go

```diff
@@ -62,7 +59,8 @@ provided that:
 
 	// Cosmos flags
 	cmd.Flags().String(cosmosflags.FlagKeyringBackend, "", "Select keyring's backend (os|file|kwallet|pass|test)")
-	cmd.Flags().String(cosmosflags.FlagNode, omittedDefaultFlagValue, "registering the default cosmos node flag; needed to initialize the cosmostx and query contexts correctly and uses flagQueryNodeUrl underneath")
+	cmd.Flags().
+		String(cosmosflags.FlagNode, omittedDefaultFlagValue, "registering the default cosmos node flag; needed to initialize the cosmostx and query contexts correctly and uses flagQueryNodeUrl underneath")
```

Comment:
I think the part about "uses flagQueryNodeUrl underneath" is ambiguous and therefore, potentially misleading. If we're going to mention this connection, let's provide more detail on "how" it uses the flag; e.g.: "sets the value of flagQueryNodeUrl internally for use by cosmos-sdk API".